pyof/v0x01/controller2switch/stats_reply.py

```diff
@@ -31,6 +35,24 @@ def __init__(self, xid=None, body_type=None, flags=None, body=b''):
         self.flags = flags
         self.body = body
 
+    def pack(self):
```

Comment:
Our pack methods signature contains a `value` argument. Please, fix this to keep the method signature consistent.